knowledge-base/scheduler-display-allday.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Display Only All-Day Appointments in the Scheduler
+description: Learn how to customize your Scheduler for Blazor to exclusively display all-day events in a multiday timeline view, ensuring optimal scheduling clarity.
+type: how-to
+page_title: Display Only All-Day Appointments in the Scheduler
+slug: scheduler-display-allday
+tags: scheduler, timeline, all-day
+ticketid: 1576096
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Scheduler for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+This KB article answers the following questions:
+
+* How to display only all-day events in the Scheduler?
+* How to show only all-day appointments in a [Timeline View]({%slug scheduler-views-timeline%}) of the Scheduler?
+* How to create a multiday scheduler view that only displays the all-day events?
+
+## Solution
+
+To achieve the desired outcome, ensure that you apply in your configuration the following points:
+
+1. Use Timeline View of the `Scheduler`.
+2. Set `SlotDuration` parameter of the `SchedulerTimelineView` to 1440 minutes (one day).
+3. Set the `IsAllDay` property of the appointment model to `true`.
+
+>caption Scheduler with a Timeline View displaying only all-day appointments
+
+````CSHTML
+// A multiday scheduler that displays only the all-day events
+<TelerikScheduler Data="@Appointments" @bind-Date="@StartDate" Width="1000px">
+    <SchedulerViews>
+        <SchedulerTimelineView StartTime="@DayStart"
+                               EndTime="@DayEnd"
+                               NumberOfDays="10"
+                               SlotDuration="1440" // minutes equals to one day / 24 hours
+                               WorkDayStart="@WorkDayStart"
+                               WorkDayEnd="@WorkDayEnd" />
+    </SchedulerViews>
+</TelerikScheduler>
+
+@code {
+    public DateTime StartDate { get; set; } = new DateTime(2019, 12, 2);
+    //the time portions are important
+    public DateTime DayStart { get; set; } = new DateTime(2000, 1, 1, 8, 0, 0);
+    public DateTime DayEnd { get; set; } = new DateTime(2000, 1, 1, 20, 0, 0);
+    public DateTime WorkDayStart { get; set; } = new DateTime(2000, 1, 1, 9, 0, 0);
+    public DateTime WorkDayEnd { get; set; } = new DateTime(2000, 1, 1, 17, 0, 0);
+    List<SchedulerAppointment> Appointments = new List<SchedulerAppointment>()
+    {
+            new SchedulerAppointment
+            {
+                Title = "Trip to Hawaii",
+                IsAllDay = true, // set to true to show as an all-day appointment 
```

Comment:
You can use single quotes (') to surround values and other elements of code. This helps improve the readability of the comments in the code samples.
```suggestion
                IsAllDay = true, // set to 'true' to show as an all-day appointment 
```